### ISSUE TITLE
EPIC: Harmonising HPO chemical phenotypes

### ISSUE THREAD

**Issue by matentzn** [obophenotype/human-phenotype-ontology#10391]
1. Strategy is described [here](https://docs.google.com/document/d/1YDIR9yKME_Jol4KSGc-oibfih3Y_miUxZ4xKco90ETI/edit#heading=h.14mpx6a8znb7)
2. Curation is prepared [here](https://docs.google.com/spreadsheets/d/1TwsHHrEDM8cTDZ6meFBgIbeeyHoumSB4CUyoauUrPSA/edit#gid=1476288483)

@LCCarmody let me know if this works for you. I would suggest:

1. Go through curation list I prepared in fill in column `chemical_phenotype`, basically saying "this is indeed a chemical phenotype" - there may be quite a few wrong ones.
2. Go through list again and pick `correct_pattern`. 
3. Once this is done, we meet and make a strategy to harmonise everything

what do you think?

**Comment by LCCarmody** [comment]
Sorry for the delay (although I knew you were off)... overall, it looks good. I assume I can add terms to the list? I think this should be done in conjunction with Protege.

**Comment by LCCarmody** [comment]
And an additional comment as I am going through this, what is 'defined_class_label'? and how is it different from hpo_label?

Second: Could we have a way to add synonyms? One issue is that there is not consistency of labels, but adding synonyms could help that situation. It could also help decrease duplicate creation before the fact.

**Comment by matentzn** [comment]
@LCCarmody this is amazing work so far!!! Beautiful!

`defined_class_label` is the lingo used in DOSDP - its the same as `hpo_label`. You can hide this column for now if it annoys you - we will add it back when we start experimenting!

> Second: Could we have a way to add synonyms? 

Unless they are very specific synonyms, synonyms should _not_ be manually curated. They should be added to the DOSDP pattern, and DOSDP tools will add them automatically. Does this make sense?

**Comment by LCCarmody** [comment]
Done with categorizing the patterns. Found one duplicate (will create ticket). What's next? Definitely would like some help pulling terms (protein/chemical).

**Comment by pnrobinson** [comment]
@LCCarmody 
Could we summarize progress to date and discuss prior to transition date?

**Comment by LCCarmody** [comment]
Yes! I am currently working through the current batches of patternized terms. Should we set up a time later this week? Early next? with or without @matentzn ?
